src/components/Carousel/Carousel.jsx

```diff
@@ -0,0 +1,23 @@
+import "./carousel.css";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import Slider from "react-slick";
```

Comment:
this was new for me, I will try this react-slick npm

Reply:
> this was new for me, I will try this react-slick npm

Thankyou so much for reviewing, I'm glad that it helped!!